wordle_solver/word_initializers/english_words_initilalizer.py

```diff
@@ -2,8 +2,9 @@
 from english_words import english_words_lower_alpha_set
 from json import loads
 from string import ascii_lowercase
+from os import path
 
-from word_initializers.word_initializer import WordInitializer
+from .word_initializer import WordInitializer
```

Comment:
Possibly better version:
from .word_initializer import WordInitializer
My PyCharm claims:
ImportError: attempted relative import with no known parent package
For similar lines. (See wordle_solver.py)